packages/prettier/package.json

```diff
@@ -8,5 +8,8 @@
   "packageManager": "yarn@3.6.3",
   "peerDependencies": {
     "prettier": ">=3.0.0"
+  },
+  "devDependencies": {
+    "@trivago/prettier-plugin-sort-imports": "^4.2.0"
```

Comment:
I think this should be a peer dependency? 

https://github.com/npm/npm/blob/2e3776bf5676bc24fec6239a3420f377fe98acde/doc/files/package.json.md#peerdependencies